pkg/gameservers/controller.go

```diff
@@ -319,20 +328,20 @@ func (c *Controller) Run(workers int, stop <-chan struct{}) error {
 // syncGameServer synchronises the Pods for the GameServers.
 // and reacts to status changes that can occur through the client SDK
 func (c *Controller) syncGameServer(key string) error {
-	c.logger.WithField("key", key).Info("Synchronising")
+	c.loggerForGameServerKey(key).Info("Synchronising")
```

Comment:
Just thinking out loud, but any thoughts on whether to move these `loggerFor*` functions into logfields somehow? (centralised logger object maybe?).

May be overkill, because there doesn't seem to be much/any duplication here, but thought I'd ask the question.

Reply:
I thought about it but could not find a nice pattern that works, you need to pass the base logger, the type and the value somehow. Two of those things are typically bound to a controller so it makes sense to have a method there.